### DISCUSSION TITLE
Do we have a python signature for translation?

### DISCUSSION THREAD

**Discussion by raulgupta** [openai/whisper#196] [Q&A] [answered]
For example: What would be the python equivalent of: 

`whisper japanese.wav --language Japanese --task translate`

Thanks!

**Comment by jongwook** [comment, marked as answer]
The following would be equivalent (realizing the function's name may be causing confusion):

```python
import whisper

model = whisper.load_model("small")
result = model.transcribe("japanese.wav", language="Japanese", task="translate", beam_size=5, best_of=5)
```

The last two arguments are just to match the default arguments in the command line, and larger model than `small` will be more reliable especially for translations.
